__tests__/commands/_helpers.js

```diff
@@ -81,9 +81,21 @@ export async function run<T, R>(
 
   let cwd;
   if (fixturesLoc) {
-    const dir = path.join(fixturesLoc, name);
+    let parentDir = name;
+    let childDir = '';
+
+    // if the name is a path, copy the parent dir, then use the child dir as the cwd
+    // there's some existing tests which reference fixtures in a parent dir (../), so excluding those
+    const nameSepIndex = name.indexOf('/');
+    if (nameSepIndex !== -1 && name.indexOf('../') === -1) {
```

Comment:
Yeah, having an object with `cwd` field like you proposed makes sense, let's give it a try